docs/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+import sys
+
+import atldld
+
+# sys.path.insert(0, os.path.abspath('.'))
+print(sys.path)
```

Comment:
I guess this is copied from `atlalign`? It prints this list of paths every time you run `make`.

By me we can remove this print, unless you think it's useful for something. Then we can leave it.